src/components/btn/btn.scss

```diff
@@ -71,29 +52,39 @@
 .btn-blue-outline {
   border: 4px solid $main-font-color;
   background-color: $primary-background-blue;
-  color: $main-font-color !important;
-  padding: 0 !important;
-  font-weight: 400 !important;
-  font-size: 32px !important;
+  color: $main-font-color;
+  padding: 0;
+  font-weight: 400;
+  font-size: 32px;
   width: 350px;
-  height: 85px !important;
+  height: 85px;
 }
 
 .btn-leave-game {
```

Comment:
please add `btn-rounded` with border radius and use it with .btn-pink-solid class and remove this class